manifests/templates/release/olm/cdi-subscription.yaml.in

```diff
@@ -7,4 +7,4 @@ spec:
   channel: beta
   name: cdi
   source: cdi
-  sourceNamespace: olm 
+  sourceNamespace: openshift-operator-lifecycle-manager 
```

Comment:
Shouldn't upstream (olm) be the default?  Maybe we can generate two of these?  One for upstream and one for down?

Reply:
I split manifests of catalogsource and subscription to k8s and os directory respectfully.